recipes/pypardiso/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pypardiso" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haasad/PyPardisoProject/archive/refs/tags/v0.2.2.tar.gz
+  sha256: b1d3a139f8f04c74f8b2b5c4b55434e1f58b00ae3fe0ca975115971906d93eac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - numpy
```

Comment:
```suggestion
```

Reply:
Sorry, the distinction between host and build is still a bit fuzzy for me. I am worried that this will somehow break things, but we can commit it and try...